Makefile

```diff
@@ -61,17 +63,17 @@ clear:
 	cargo clean
 	$(CLEAN_FFI_DIR)
 
-lib-native: Cargo.lock wgpu-native/Cargo.toml $(wildcard wgpu-native/**/*.rs)
+lib-native: Cargo.lock wgpu-native/Cargo.toml $(WILDCARD_WGPU_NATIVE)
 	cargo build --manifest-path wgpu-native/Cargo.toml --features "local,$(FEATURE_NATIVE)"
 
-lib-remote: Cargo.lock wgpu-remote/Cargo.toml $(wildcard wgpu-native/**/*.rs wgpu-remote/**/*.rs)
+lib-remote: Cargo.lock wgpu-remote/Cargo.toml $(WILDCARD_WGPU_NATIVE_AND_REMOTE)
 	cargo build --manifest-path wgpu-remote/Cargo.toml --features $(FEATURE_RUST)
 
-ffi/wgpu.h: wgpu-native/cbindgen.toml $(wildcard wgpu-native/**/*.rs)
+ffi-wgpu: wgpu-native/cbindgen.toml $(WILDCARD_WGPU_NATIVE)
```

Comment:
this is no longer a refactor, this is changing the dependencies, since Makefile does no longer know when it needs to rebuild the header, and when it's up-to-date.
Let's perhaps leave the `"ffi/somehing.h" in the targets?

Reply:
Oh I was not aware that the command itself was used for the rebuilding of the header. Well then, I will change that back to how it was and only pushing the variables `WILDCARD_WGPU_NATIVE` & `WILDCARD_WGPU_NATIVE_AND_REMOTE`